### ISSUE TITLE
Timescaledb 1.7.1, PostgreSQL 12.3 Windows startup error

### ISSUE THREAD

**Issue by cristihcd** [timescale/timescaledb#1924]
**Relevant system information:**
 - OS: Windows 10 x64
 - PostgreSQL version: postgres (PostgreSQL) 12.3
 - TimescaleDB version: 1.7.1
 - Installation method: zip file (Windows)
**Describe the bug**
PostgreSQL fails to start with TimescaleDB 1.7.1. The error is:
`FATAL:  could not load library "c:/pgsql/12/lib/timescaledb.dll": The specified procedure could not be found.`
The problem does not appear on the same machine with **TimescaleDB 1.7.0**.

**To Reproduce**
1. Download timescaledb from https://timescalereleases.blob.core.windows.net/windows/timescaledb-postgresql-12_1.7.1-windows-amd64.zip
2. Launch setup, tune
3. Launch server

**Additional context**
PostgreSQL starts ok without the extension in shared_preload_libraries.
The same version of PostgreSQL starts ok with TimescaleDB 1.7.0

I compared the bin, lib and share/extension folders and they are identical between PostgreSQL 12.3 with TimescaleDB 1.7.0 and PostgreSQL 12.3 with TimescaleDB 1.7.1
(with the exception of name/size differences between 1.7.0 and 1.7.1 files).

Dependencies screenshots:
![image](https://user-images.githubusercontent.com/5652964/82426064-08184d80-9a90-11ea-911d-8c1de12af637.png)

After copying timescaledb.dll, timescaledb-1.7.1.dll and timescaledb-tsl-1.7.1.dll to the bin folder in order to see the dependencies:
![image](https://user-images.githubusercontent.com/5652964/82426371-6e04d500-9a90-11ea-843e-5fdae5e24a2b.png)

**Comment by k-rus** [comment]
@cristihcd Thank you for reporting.
Have you run the setup as Administrator or non-elevated? What is the output of running the setup?
Do I understand correctly that you manually copied the binaries into the pg lib folder?

**Comment by k-rus** [comment]
@cristihcd I am wonder if you were able to run setup.exe again with Administrative preventable and fix the installation. Or can you provide more information about your exeprience?

**Comment by cristihcd** [comment]
Hello,

I downloaded the zip file again from the documentation page and everything went well. 

It turns out the copy I had downloaded was somehow corrupted. I had downloaded it by modifying the URL from (...1.7.1...) to (...1.7.2...), since the documentation page was not yet updated.

The sha1 for the original (broken) dll's in the package:
d71dfb2f769b09dd27ce56866cc23f8110537cfc *timescaledb.dll
b6fb0ee67a3309a20bea9365f10fb38ea938a709 *timescaledb-1.7.1.dll
8fa9f88b7c4d2735f55df400f9f287e84b455020 *timescaledb-tsl-1.7.1.dll

Thanks for looking into this.

@k-rus I used the zip files, no Administrator privileges. I unzipped PostgreSQL, then ran setup.exe from the Timescaledb zip.
